### DISCUSSION TITLE
Wemos d1 mini

### DISCUSSION THREAD

**Discussion by gohigher14** [music-assistant/support#989] [Q&A] [answered]
Hi,

Can i use just a wemos d1 mini as a media player? Is this one supported?

**Comment by OzGav** [comment, marked as answer]
Do some Google searches for this. Whilst ESPHOME has support that doesn't mean any board is suitable. https://www.youtube.com/watch?v=SEH-DxOsywg

**Reply by gohigher14** [reply]
> Do some Google searches for this. Whilst ESPHOME has support that doesn't mean any board is suitable. https://www.youtube.com/watch?v=SEH-DxOsywg

Ok thanks. But isn't there a list of all esp boards that do work? Sorry very new to this and still learning. Just don't want to buy a board that i already have at home.

**Reply by OzGav** [reply]
You should watch the video I linked but here is a list https://esphome.github.io/media-players/
